server/utils/stringUtils.ts

```diff
@@ -1,10 +1,24 @@
 export default class StringUtils {
+  static convertFromUrlSlug(slug: string): string {
+    return slug
+      .split('-')
+      .map(word => this.properCase(word))
+      .join(' ')
+  }
+
   static convertToTitleCase(sentence: string | null): string {
     return sentence === null || StringUtils.isBlank(sentence)
       ? ''
       : sentence.split(' ').map(StringUtils.properCaseName).join(' ')
   }
 
+  static convertToUrlSlug(string: string): string {
+    return string
```

Comment:
Alternatively, this can be simplified just to use a single `.replace`, like [this](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/385/files#diff-86e73c78364b077353b0b89bccbd849eec5218bc84c18e97fdd46c1b51bfa171R11)/[this](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/385/files#diff-763d9554d0518fa888472873a2ed51d4d58e75d9856760c401bb2264ba65d233R239), although if Slug does more than replace one character with another, it might be useful?

Reply:
Replace is probably suitable for now.  Bearing in mind whatever we strip out of the course name, has to be put back in when we search for it, unless the api returned a slug/url friendly ID?

Reply:
Yeah, I do wonder if storing this in the API is a more future-proof solution here, the endpoint could return a unique list of course names and slugs for use here, rather than the full courses